torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -867,6 +867,14 @@ RegisterOperators reg(
            }
            return 0;
          }),
+     Operator(
+         "prim::Bottom() -> Bottom",
+         [](Stack& stack) {
+           // Todo: add special unitialized ivalue that will error
+           // whenever it is used
+           push(stack, 1);
```

Comment:
Pushing 1 will lead to incredibly confusing errors. None would be a less surprising choice.